src/test/java/it/gov/pagopa/pu/bff/service/organization/BrokerServiceImplTest.java

```diff
@@ -0,0 +1,111 @@
+package it.gov.pagopa.pu.bff.service.organization;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import it.gov.pagopa.pu.bff.config.DefaultConfigFe;
+import it.gov.pagopa.pu.bff.connector.OrganizationClientImpl;
+import it.gov.pagopa.pu.bff.dto.generated.ConfigFE;
+import it.gov.pagopa.pu.bff.mapper.PersonalisationFE2ConfigFEMapper;
+import it.gov.pagopa.pu.bff.util.TestUtils;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.EntityModelBroker;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.EntityModelOrganization;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.PersonalisationFe;
+import it.gov.pagopa.pu.p4paauth.model.generated.UserInfo;
+import it.gov.pagopa.pu.p4paauth.model.generated.UserOrganizationRoles;
+import java.util.Collection;
+import java.util.Collections;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+@ExtendWith(MockitoExtension.class)
+class BrokerServiceImplTest {
+
+  @Mock
+  private OrganizationClientImpl organizationClientMock;
+
+  @Mock
+  private DefaultConfigFe defaultConfigFeMock;
+
+  @Mock
+  private PersonalisationFE2ConfigFEMapper personalisationFE2ConfigFEMapperMock;
+
+  @InjectMocks
+  private BrokerServiceImpl brokerServiceMock;
```

Comment:
This is not a mock, this is the tested class

Reply:
updated